docs/index.md

```diff
@@ -0,0 +1,49 @@
+---
+id: introduction
+title: Introduction
+---
+
+Podium is a library for building [micro frontends](https://micro-frontends.org/).
```

Comment:
Would it make sense to update this to include Island Architectures as well, which is a more recent name for more or less the same thing?
We could link to this article which coined the term: https://jasonformat.com/islands-architecture/

Reply:
I'd love to work that in, especially when talking about client-side assets and isolation. I think I'll save it for the next PR though.

Reply:
good idea, as it make sense to have something more concrete than just saying "oh you can make X with Podium".